### PR TITLE
Release/2022 03 24 grey wolf

### PR DESCRIPTION
**Pull request by fbwoolf**
> Try out this version of the Hiro Wallet - download [extension builds](https://github.com/hirosystems/stacks-wallet-web/actions/runs/2041240623).

Content:

- https://github.com/hirosystems/stacks-wallet-web/pull/2312
- https://github.com/hirosystems/stacks-wallet-web/pull/2277
- https://github.com/hirosystems/stacks-wallet-web/pull/2313
- https://github.com/hirosystems/stacks-wallet-web/pull/2304

cc/ @kyranjamie @fbwoolf @beguene @He1DAr

### CODE REVIEW

**Comment by fbwoolf** [comment]
We need to make sure to switch the `hiro-config` file here to point to 'main' before merging.

File: src/app/query/hiro-config/hiro-config.query.ts [hunk 1]
```diff
@@ -19,17 +19,19 @@ interface ActiveFiatProviderType {
...
 
 interface HiroConfig {
   messages: any;
   activeFiatProviders?: Record<string, ActiveFiatProviderType>;
-  feeEstimations?: FeeEstimationsConfig;
+  feeEstimationsMinMax?: FeeEstimationsConfig;
 }
 
-const GITHUB_PRIMARY_BRANCH = 'dev';
+const GITHUB_PRIMARY_BRANCH = 'dev'; // TODO: Change to main
```
Comment (fbwoolf) [inline src/app/query/hiro-config/hiro-config.query.ts]: We need to make sure to switch this to `main` before merging.

**Review by landitus — APPROVED** [review]
LGTM!

**Comment by fbwoolf** [comment]
This was found while testing the new home page and onboarding steps PR. It seems unrelated to those changes, but we should confirm this exists in production before shipping this release: https://github.com/hirosystems/stacks-wallet-web/issues/2328

Also, we should retest here to confirm the issue still exists.

**Review by kyranjamie — APPROVED** [review]

**Comment by fbwoolf** [comment]
Note: The `GITHUB_PRIMARY_BRANCH` on the latest build is set to `main` which is a breaking change. If we need to test this again we will have to test a previous build or change it back to `dev` temporarily.

**Comment by fbwoolf** [comment]
@beguene on Monday, can you look at this issue and confirm if the buy options are working as you expect: https://github.com/hirosystems/stacks-wallet-web/issues/2328

*https://github.com/hirosystems/stacks-wallet-web/pull/2332#issuecomment-1079231383

**Comment by beguene** [comment]
FYI This https://github.com/hirosystems/stacks-wallet-web/pull/2313 should have been released before merging
https://github.com/hirosystems/stacks-wallet-web/pull/2312 as I mentioned here https://github.com/hirosystems/stacks-wallet-web/pull/2313#issue-1173464120
This is causing this issue https://github.com/hirosystems/stacks-wallet-web/issues/2328
If we release this grey-wolf PR it will fix https://github.com/hirosystems/stacks-wallet-web/issues/2328

**Comment by beguene** [comment]
> @beguene on Monday, can you look at this issue and confirm if the buy options are working as you expect: #2328
> 
> *[#2332 (comment)](https://github.com/hirosystems/stacks-wallet-web/pull/2332#issuecomment-1079231383)

Thanks @fbwoolf. We can simply release everything in this grey wolf PR and the issue https://github.com/hirosystems/stacks-wallet-web/issues/2328 will be solved.

**Comment by fbwoolf** [comment]
> FYI This #2313 should have been released before merging #2312 as I mentioned here [#2313 (comment)](https://github.com/hirosystems/stacks-wallet-web/pull/2313#issue-1173464120) This is causing this issue #2328 If we release this grey-wolf PR it will fix #2328

Ok, thanks, I did see that comment but it was already merged into dev, so at that point we should have just merged it into main separately? I did think that was the issue but wanted to confirm with you. 👍  Seems like we are good to release this today! Sorry for the confusion.

**Pull request merged**